src/compiler/transformers/ts.ts

```diff
@@ -2261,7 +2243,7 @@ namespace ts {
 
             const statements: Statement[] = [];
             if (shouldEmitVarForEnumDeclaration(node)) {
-                addVarForEnumDeclaration(statements, node);
+                addVarForEnumOrModuleDeclaration(statements, node);

```

Comment:
https://github.com/Microsoft/TypeScript/pull/8064/commits/27adb8c363c6a172a5a489929251f569666bd2d2#diff-c0d0e8b1528663b6cff3bb893150cec3R2382 
Renamed ```addVarForModuleDeclaration``` to ```addVarForEnumOrModuleDeclaration```